cli/src/helpers/installDependencies.ts

```diff
@@ -7,9 +7,9 @@ import {
 } from "~/utils/getUserPkgManager.js";
 import { logger } from "~/utils/logger.js";
 
-type Options = {
+interface Options {
```

Comment:
https://v6--typescript-eslint.netlify.app/rules/consistent-type-definitions is the rule to enforce which of `interface` or `type` you prefer to use consistently. In v6 right now it's moved from _`strict`_ to _`stylistic`_. How does that make you feel?

For reference, a couple of full text regex searches on the `next` branch:

* `interface .+ \{`: 44 results in 40 files
* `type .+ = \{`: 14 results in 12 files

Reply:
this is one of the few that i don't feel strongly about.

my personal preference is "type by default, interface only if necessary (ie need to extend)" but judging by these stats that's not actually what we're doing currently.

what's the eslint team's opinion on this?

Reply:
We in typescript-eslint[^1] have `interface` over `type` as the default in our recommended rulesets, but I don't believe any of us feel extremely strongly about it. I personally don't. IMO the most important thing for a codebase is to standardize on one to reduce confusion.

I personally have leaned towards `interface`s because they've historically had better error messages in edge cases than `type`. Though over the last couple of years, TypeScript has gotten a lot better at using names for types. So that's not as powerful a motivator as it used to be...

[^1]: Sorry to nitpick - we're a separate team from ESLint core and don't want to accidentally give the impression we're speaking for them 😅

Reply:
Personally, I think `interface` is superior. The official TypeScript wiki recommends preferring it because of performance options and the ability to augment types.

Reply:
Just to be clear - would you like me to disable the rule and revert its changes? I'm happy to, just not sure from the discussion 😅